diff -up phonon-4.11.1/phonon/experimental/backendcapabilities.cpp.omv~ phonon-4.11.1/phonon/experimental/backendcapabilities.cpp
--- phonon-4.11.1/phonon/experimental/backendcapabilities.cpp.omv~	2023-04-19 01:52:29.076080191 +0200
+++ phonon-4.11.1/phonon/experimental/backendcapabilities.cpp	2023-04-19 01:53:09.380466527 +0200
@@ -23,6 +23,9 @@
 #include "../backendcapabilities.h"
 #include "globalconfig.h"
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wenum-constexpr-conversion"
+
 namespace Phonon
 {
 namespace Experimental
@@ -45,3 +48,5 @@ QList<VideoCaptureDevice> BackendCapabil
 
 } // namespace Experimental
 } // namespace Phonon
+
+#pragma GCC diagnostic pop
diff -up phonon-4.11.1/phonon/experimental/objectdescription.h.omv~ phonon-4.11.1/phonon/experimental/objectdescription.h
--- phonon-4.11.1/phonon/experimental/objectdescription.h.omv~	2023-04-19 01:47:08.161515326 +0200
+++ phonon-4.11.1/phonon/experimental/objectdescription.h	2023-04-19 01:49:37.586298669 +0200
@@ -25,6 +25,9 @@
 
 #include "../objectdescription.h"
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wenum-constexpr-conversion"
+
 namespace Phonon
 {
 namespace Experimental
@@ -55,4 +58,6 @@ Phonon::VideoCaptureDevice phononExperim
 } // namespace Experimental
 } // namespace Phonon
 
+#pragma GCC diagnostic pop
+
 #endif // PHONON_EXPERIMENTAL_OBJECTDESCRIPTION_H
diff -up phonon-4.11.1/phonon/experimental/visualization.h.omv~ phonon-4.11.1/phonon/experimental/visualization.h
--- phonon-4.11.1/phonon/experimental/visualization.h.omv~	2023-04-19 01:51:35.748551549 +0200
+++ phonon-4.11.1/phonon/experimental/visualization.h	2023-04-19 01:52:11.957912747 +0200
@@ -30,6 +30,9 @@
 #include "../objectdescriptionmodel.h"
 #include "../medianode.h"
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wenum-constexpr-conversion"
+
 namespace Phonon
 {
 class AudioPath;
@@ -103,5 +106,7 @@ class PHONONEXPERIMENTAL_EXPORT Visualiz
 } // namespace Experimental
 } // namespace Phonon
 
+#pragma GCC diagnostic pop
+
 #endif // PHONON_VISUALIZATION_H
 // vim: sw=4 ts=4 tw=80
diff -up phonon-4.11.1/phonon/objectdescription.h.omv~ phonon-4.11.1/phonon/objectdescription.h
--- phonon-4.11.1/phonon/objectdescription.h.omv~	2023-04-19 01:48:33.321562237 +0200
+++ phonon-4.11.1/phonon/objectdescription.h	2023-04-19 01:49:14.182035383 +0200
@@ -32,6 +32,8 @@
 #include <QtCore/QString>
 #include <QtCore/QVariant>
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wenum-constexpr-conversion"
 
 namespace Phonon
 {
@@ -388,4 +390,5 @@ Q_DECLARE_METATYPE(Phonon::DeviceAccessL
 
 
 
+#pragma GCC diagnostic pop
 #endif // PHONON_OBJECTDESCRIPTION_H
