diff -urN phonon-4.7.0/PhononConfig.cmake.in phonon-4.7.0-patched/PhononConfig.cmake.in
--- phonon-4.7.0/PhononConfig.cmake.in	2013-11-05 03:49:06.000000000 +1100
+++ phonon-4.7.0-patched/PhononConfig.cmake.in	2013-11-06 13:50:11.583719359 +1100
@@ -14,7 +14,7 @@
 set(PHONON_INCLUDE_DIR "${rootDir}/@INCLUDE_INSTALL_DIR@")
 
 set(PHONON_LIBRARY_DIR "${rootDir}/@LIB_INSTALL_DIR@")
-set(PHONON_BUILDSYSTEM_DIR "${rootDir}/@BUILDSYSTEM_INSTALL_DIR@")
+set(PHONON_BUILDSYSTEM_DIR "${rootDir}/usr/@BUILDSYSTEM_INSTALL_DIR@")
 set(PHONON_LIB_SONAME "@PHONON_LIB_SONAME@")
 
 if(NOT TARGET Phonon::@PHONON_LIB_SONAME@)
